spec/controllers/v0/virtual_agent_token_controller_spec.rb

```diff
@@ -6,26 +6,42 @@
 RSpec.describe V0::VirtualAgentTokenController, type: :controller do
   describe '#create' do
     context 'when external service is healthy' do
-      let(:recorded_token) do
-        'ew0KICAiYWxnIjogIlJTMjU2IiwNCiAgImtpZCI6ICJsY0oxTXFpNkdKYXdCZEw5Y0dieEt5S1R6OE0iLA0KICAieDV0IjogImxjSjFNcWk2' \
```

Comment:
Is this the actual token? A little concerned about a credential leak

Reply:
No, it's an expired non-secret token